.github/workflows/build-and-test.yml

```diff
@@ -112,6 +112,31 @@ jobs:
           ~/actionutils.sh wait_for_osds 3
           sudo microceph.ceph -s
 
+      - name: Add and relate RadosGW charm
+        run: |
+          set -eux
+          prev=$(juju ssh microceph/0 -- "sudo microceph.ceph osd pool ls | wc -l")
+          juju deploy -m admin/controller ch:ceph-radosgw
+          sleep 120
+          juju integrate -m admin/controller ceph-radosgw microceph
+          sleep 420
+          for i in $(seq 1 20); do
+            curr=$(juju ssh microceph/0 -- "sudo microceph.ceph osd pool ls | wc -l")
+            if [[ $curr -gt $prev ]]; then
```

Comment:
Hmmm... in the test run for this commit I can see it passing the test when changing from 0 to 1 pools. I have a bad feeling this might again be the `.mgr` pool; in any case 1 pool seems low for radosgw, it should at least create pools for rgw.log, rgw.meta rgw.control I think. Maybe we should look pools matching rgw to make sure

```
 + [[ 1 -gt 0 ]]
+ echo 'Found new pools'
+ break
```

Reply:
Done.